test/fixtures/replace-global-define/expected.js

```diff
@@ -0,0 +1,13 @@
+System.registerDynamic([], true, function ($__require, exports, module) {
+  var define,
+      global = this || self,
+      GLOBAL = global;
+  if (undefined) {
```

Comment:
Yes, indeed this could be used for dead code elimintation.  But i've forgotten an important code change: the top-level `var define` statement shouldn't be necessary anymore.

For more context: https://github.com/systemjs/babel-plugin-transform-cjs-system-wrapper/issues/7